bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/robot/RRMapFileParser.java

```diff
@@ -299,6 +300,16 @@ public String toString() {
         return sw.toString();
     }
 
+    private void printAreaDetails(ArrayList<float[]> areas, PrintWriter pw) {
+        areas.forEach(area -> {
+            pw.printf("\tArea details:");
+            for (int i = 0; i < area.length; i++) {
+                pw.printf("\t%.1f", area[i]);
+            }
+            pw.print("\r\n");
```

Comment:
I think you should not directly write new lines, Just as done above use:
```suggestion
            pw.println();
```

Reply:
yes, agree. done